docs/advanced/api-v2/rest.md

```diff
@@ -0,0 +1,4019 @@
+                }
+            ]
+        }
+    ```
+
+### Get Block [GET `/blocks/{block_index}`]
+
+Return the information of a block
+
++ Parameters
+    + block_index: `840464` (int, required) - The index of the block to return 
+
++ Response 200 (application/json)
+
+    ```
+        {
+            "result": {
+                "block_index": 840464,
+                "block_hash": "00000000000000000001093d4d6b21b80800fff6e5ea15cce6d65066f482cce9",
+                "block_time": 1713852783,
+                "previous_block_hash": "00000000000000000002db1e5aa19784eec3de949f98ec757e7a7f2fc392079d",
+                "difficulty": 86388558925171.02,
+                "ledger_hash": "b3f8cbb50b0705a5c4a8495f8b5128de13a32daebd8ac5e8316a010f0d203584",
+                "txlist_hash": "84bdc5b9073f775a2b65de7da2b10b89a2235f3501883b0a836e41e68cd00d46",
+                "messages_hash": "801d961c45a257f85ef0f10a6a8fdf048a520ae4861c0903f26365b3eaaaf540"
+            }
+        }
+    ```
+
+### Get Transactions By Block [GET `/blocks/{block_index}/transactions`]
+
+Returns the transactions of a block
+
++ Parameters
+    + block_index: `840464` (int, required) - The index of the block to return 
+
++ Response 200 (application/json)
+
+    ```
+        {
+            "result": [
+                {
+                    "tx_index": 2726605,
+                    "tx_hash": "876a6cfbd4aa22ba4fa85c2e1953a1c66649468a43a961ad16ea4d5329e3e4c5",
+                    "block_index": 840464,
+                    "block_hash": "00000000000000000001093d4d6b21b80800fff6e5ea15cce6d65066f482cce9",
+                    "block_time": 1713852783,
+                    "source": "178etygrwEeeyQso9we85rUqYZbkiqzL4A",
+                    "destination": "",
+                    "btc_amount": 0,
+                    "fee": 56565,
+                    "data": "16010b9142801429a60000000000000001000000554e4e45474f544941424c45205745204d555354204245434f4d4520554e4e45474f544941424c4520574520415245",
+                    "supported": 1
+                }
+            ]
+        }
+    ```
+
+### Get Events By Block [GET `/blocks/{block_index}/events`]
+
+Returns the events of a block
+
++ Parameters
+    + block_index: `840464` (int, required) - The index of the block to return 
+
++ Response 200 (application/json)
+
+    ```
+        {
+            "result": [
+                {
+                    "event_index": 14194760,
+                    "event": "BLOCK_PARSED",
+                    "bindings": {
+                        "block_index": 840464,
+                        "ledger_hash": "b3f8cbb50b0705a5c4a8495f8b5128de13a32daebd8ac5e8316a010f0d203584",
+                        "messages_hash": "801d961c45a257f85ef0f10a6a8fdf048a520ae4861c0903f26365b3eaaaf540",
+                        "txlist_hash": "84bdc5b9073f775a2b65de7da2b10b89a2235f3501883b0a836e41e68cd00d46"
+                    },
+                    "block_index": 840464,
```

Comment:
I'd skip `block_index` and `timestamp`. The index is in the query, and both are the same for every entry.

Reply:
done